### DISCUSSION TITLE
New Component: Badge

### DISCUSSION THREAD

**Discussion by WilferCiro** [heroui-inc/heroui#312] [Feature requests]
I would like to suggest the Badge component.

This component is very important for ecommerce, allowing to add labels to products and the quantity of items in the shopping cart icon, In addition, as a graphic element, it can be important in other types of pages.

**Additional context**

Examples from antd

![image](https://user-images.githubusercontent.com/14842306/154745286-3aefb2f9-5741-4955-ad68-70291003d962.png)

![image](https://user-images.githubusercontent.com/14842306/154745801-c2b678fe-85ec-4064-a02d-27ca757278e0.png)

**Comment by wingkwong** [comment]
https://www.heroui.com/docs/components/badge
